package-lock.json

```diff
@@ -9649,4 +9649,4 @@
       }
     }
   }
-}
+}
```

Comment:
I accidentally pushed some changes my IDE made to this file, and then undid them manually, but for some reason this is still in the diff. Feel free to not merge this file

Reply:
no big deal. probably extra empty line at the end of file